app/lib/scorecard/models.dart

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:gcloud/db.dart' as db;
+import 'package:meta/meta.dart';
+
+import 'package:pub_dartlang_org/search/scoring.dart'
+    show calculateOverallScore;
+
+import '../shared/model_properties.dart';
+
+String _id(String packageName, String packageVersion) =>
+    '$packageName/$packageVersion';
+
+final _gzipCodec = new GZipCodec();
+
+/// Summary of various reports for a given PackageVersion.
+@db.Kind(name: 'ScoreCard', idType: db.IdType.String)
+class ScoreCard extends db.ExpandoModel {
+  @db.StringProperty(required: true)
+  String packageName;
+
+  @db.StringProperty(required: true)
+  String packageVersion;
+
+  @db.DateTimeProperty(required: true)
+  DateTime packageCreated;
+
+  @db.DateTimeProperty(required: true)
+  DateTime packageVersionCreated;
+
+  @db.BoolProperty()
+  bool isDiscontinued;
+
+  @CompatibleStringListProperty()
+  List<String> panaPlatformTags;
+
+  @db.DoubleProperty()
+  double documentationScore;
```

Comment:
Document the fields in this key data class? For instance, is the values in documentationScore between 0.0 and 1.0, or 0.0 and 100.0?